testing/regression/cypress/step_definitions/hl7/generate-mark-reviewed.steps.js

```diff
@@ -133,65 +171,61 @@ When("I Generate HL7 messages to api and mark as review", () => {
                   "clientsecret": clientsecret
                 }
               }).then((response) => {                  
-                expect(response.status).to.eq(200);                                   
+                expect(response.status).to.eq(200);                              
                 if (response.body.nbsInfo.nbsInterfaceStatus === "QUEUED" || response.body.nbsInfo.nbsInterfacePipeLineStatus === "IN PROGRESS") {                  
                   NBSresponse = response.body;                  
                   cy.log(NBSresponse.nbsInfo.nbsInterfaceStatus);
                   cy.wait(20000);
                   checkStatusRequest();   
                 } else if(response.body.nbsInfo.nbsInterfaceStatus === "Success" && response.body.nbsInfo.nbsInterfacePipeLineStatus === "COMPLETED") {
-                  
-                  cy.visit("https://app.int1.nbspreview.com/nbs/HomePage.do?method=loadHomePage");
+                  let fakeSSN = Cypress.env().fakeSSN;
+                  let fakeFullName = Cypress.env().fakeFullName;
+                  cy.wait(1000);
+                  cy.log(response.body.nbsIngestionInfo[0].logComment);
+                  cy.log(response.body.nbsIngestionInfo[1].logComment);
+                  cy.log(response.body.nbsIngestionInfo[2].logComment);
```

Comment:
Are you using this for testing or final code?

Reply:
Will clean up and remove logs added for demo.